databases/week2/library-database.sql

```diff
@@ -0,0 +1,53 @@
+use library;
```

Comment:
i am sure you created the database/schema, however its always a good idea to add the create statement as well as use statement